pkg/controller/certificates/util.go

```diff
@@ -264,13 +264,13 @@ func GenerateLocallySignedTemporaryCertificate(crt *cmapi.Certificate, pkData []
 
 // RenewBeforeExpiryDuration will return the amount of time before the given
 // NotAfter time that the certificate should be renewed.
-func RenewBeforeExpiryDuration(notBefore, notAfter time.Time, specRenewBefore *metav1.Duration) time.Duration {
-	renewBefore := cmapi.DefaultRenewBefore
+func RenewBeforeExpiryDuration(notBefore, notAfter time.Time, specRenewBefore *metav1.Duration, defaultRenewBeforeExpiryDuration time.Duration) time.Duration {
+	renewBefore := defaultRenewBeforeExpiryDuration
```

Comment:
`cmapi.DefaultRenewBefore` is still the default value of the command line flag, so current users who have not set this command line flag will not see any change in behaviour.